### PR TITLE
Fix validation of server42 ingress rules

### LINKED ISSUE #3673: Flow endpoint with server42-enabled switch has rule discrepancies
Create a flow with at least one endpoint being a server42-enabled switch.
Validate the switch
**Expected:** no rule discrepancies, all rules appear in 'proper' section
**Actual:** `EXCLUSION_FLOW` type rule appears in 'missing' section, `SERVER_42_INGRESS` type rule appears in 'excess' section

- rules are not getting synchronized via sync switch api
- also reproduces during switch update or partial update

### PR DESCRIPTION
**Pull request by rozdy**
Close #3673

### CODE REVIEW

File: src-java/kilda-model/src/main/java/org/openkilda/converters/CookieConverter.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  *   limitations under the License.
  */
 
-package org.openkilda.persistence.converters;
```
Comment (surabujin) [inline src-java/kilda-model/src/main/java/org/openkilda/converters/CookieConverter.java:16]: Why it is moved outside neo/ogm module?

File: src-java/kilda-model/src/main/java/org/openkilda/model/FlowPath.java [hunk 2]
```diff
@@ -85,7 +86,7 @@
     @Relationship(type = "owns", direction = INCOMING)
     private Flow flow;
 
-    @Convert(graphPropertyType = Long.class)
+    @Convert(CookieConverter.class)
```
Comment (surabujin) [inline src-java/kilda-model/src/main/java/org/openkilda/model/FlowPath.java:89]: What the difference?

**Review by timofei-durakov — CHANGES_REQUESTED** [review]
No unit tests, please add at least one reproducing this issue

**Comment by rozdy** [comment]
> No unit tests, please add at least one reproducing this issue

Reworked and added test.

**Review by timofei-durakov — APPROVED** [review]

**Review by niksv — APPROVED** [review]

**Review by sergii-iakovenko — APPROVED** [review]

**Pull request merged**